packages/locales/lib/human/pl.json

```diff
@@ -39,10 +39,10 @@
   "backups": "Kopie zapasowe",
   "backups_subtitle": "Umożliwia tworzenie kopii zapasowych i ładowanie ustawień na różne urządzenia",
   "badge_0": "Brak",
-  "badge_1": "Brąz",
-  "badge_2": "Srebro",
-  "badge_3": "Złoto",
-  "basic": "Podstawowy",
+  "badge_1": "Podstawowy",
```

Comment:
Please change it to `Podst.` with a dot at the end.
Otherwise this wil become too long.